Tools/scripts/nm2def.py

```diff
@@ -97,7 +98,7 @@ def main():
     exports = export_list(s)
     f = sys.stdout # open('PC/python_nt.def','w')
     f.write(DEF_TEMPLATE % (exports))
-    f.close()
+    # f.close()
```

Comment:
Why don't you close the file in this case?
```python
with sys.stdout as fp:
    fp.write(DEF_TEMPLATE % (exports))
```

Reply:
No need to close the stdout.

Actually, closing the stdout looks suspicious.